src/screens/account/ChangeInfoScreen.tsx

```diff
@@ -67,16 +75,24 @@ const ChangeInfoScreen: React.FC = () => {
   };
 
   const {dispatchUpdateInfo} = useUpdateInfo();
+  const update = useUpdateInfoUser();
   const updateInfo = () => {
     dispatchUpdateInfo({
       id: '1',
       name: name,
-      gender: gender,
-      dob: dob,
+      gender: gender === 'Khác' ? '2' : gender === 'Nam' ? '0' : '1',
+      birthday: dob,
       email: email,
       phoneNumber: phone,
-      avatar: avatar,
+      picture: avatar,
     });
+    update(
+      name,
+      dob.replace(/(\d{2})[/](\d{2})[/](\d{4})/, '$3-$2-$1'),
+      gender === 'Khác' ? 2 : gender === 'Nam' ? 0 : 1,
+      phone,
+      email,
+    );
```

Comment:
những function cần truyền vào nhiều tham số như này thì nên convert lại.
Nhận vào param là 1 object
```suggestion
    function update(params){
      const name = params.name
      ....
       userService.update(...)
    } 
    update({
      name: name,
      dob: dob.replace(/(\d{2})[/](\d{2})[/](\d{4})/, '$3-$2-$1'),
      gender: gender === 'Khác' ? 2 : gender === 'Nam' ? 0 : 1,
      phone: phone,
      email: email,
    });
```